src/oe.py

```diff
@@ -777,6 +777,8 @@ def get_os_release():
             architecture = os_release_info['LIBREELEC_ARCH']
         if 'LIBREELEC_BUILD' in os_release_info:
             build = os_release_info['LIBREELEC_BUILD']
+        if 'LIBREELEC_PROJECT' in os_release_info:
+            project = os_release_info['LIBREELEC_PROJECT']
```

Comment:
`project` should be reset at top of function:
`distribution = version = architecture = build = project = builder_name = builder_version = ''`

Reply:
fixed